openshift-with-appstudio-test/e2e/jvm_build_service_test.go

```diff
@@ -395,4 +396,57 @@ func TestExampleRun(t *testing.T) {
 			debugAndFailTest(ta, "timed out waiting for some artifactbuilds and dependencybuilds to complete")
 		}
 	})
+
+	ta.t.Run("contaminated build is resolved", func(t *testing.T) {
+		//our sample repo has Netty which is contaminated by JCTools
+		var contaminated string
+		err = wait.PollImmediate(ta.interval, 2*ta.timeout, func() (done bool, err error) {
+
+			dbContaminated := false
+			dbList, err := jvmClient.JvmbuildserviceV1alpha1().DependencyBuilds(ta.ns).List(context.TODO(), metav1.ListOptions{})
```

Comment:
log this error if it is not nil ... might help us in resource constraint situations